crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -1306,13 +1309,18 @@ where
 
         let import_result = self.inner.import_block(block, new_cache).await?;
         let (root_block_sender, root_block_receiver) = mpsc::channel(0);
+        let (block_import_throttling_sender, mut block_import_throttling_receiver) =
+            mpsc::channel(0);
 
         self.imported_block_notification_sender
             .notify(move || ImportedBlockNotification {
                 block_number,
                 root_block_sender,
+                block_import_throttling_sender,
             });
 
+        block_import_throttling_receiver.next().await;
```

Comment:
Sender can and will be cloned, thus you need to wait in a loop until you get `None`.

Reply:
I don't see the sender is ever cloned,  and only one sender is expected per https://github.com/subspace/subspace/pull/737#discussion_r939604658, did I miss something?

Reply:
https://github.com/subspace/subspace/blob/71a52e4f6588eddc06564dbedd8d17850333181d/crates/sc-consensus-subspace/src/notification.rs#L53-L54

Reply:
Right, I missed that.